src/models/item.ts

```diff
@@ -173,7 +173,7 @@ export class ItemEntity {
   }
 
   getPrinted(value: ResponseItem, context: { items: ItemEntity[]; responses: ResponseItem[] | string[] }): string {
-    const allowedTypes = ['singleSelect', 'multiSelect', 'slider', 'text']
+    const allowedTypes = ['singleSelect', 'multiSelect', 'slider', 'text', 'numberSelect']
```

Comment:
This function needs to be updated to know how to handle the `numberSelect` type

Try changing 
`} else if (this.inputType === 'text') {`
to 
`} else if (this.inputType === 'text' || this.inputType === 'numberSelect' || ) {` in line 217